predict.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""torchgeo model inference script."""
+
+import argparse
+import os
+from typing import Dict, Tuple, Type, cast
+
+import pytorch_lightning as pl
+import rasterio as rio
+import torch
+from kornia.contrib import CombineTensorPatches
+from omegaconf import OmegaConf
+
+from torchgeo.datamodules import (
+    BigEarthNetDataModule,
+    ChesapeakeCVPRDataModule,
+    COWCCountingDataModule,
+    CycloneDataModule,
+    ETCI2021DataModule,
+    EuroSATDataModule,
+    InriaAerialImageLabelingDataModule,
+    LandCoverAIDataModule,
+    NAIPChesapeakeDataModule,
+    OSCDDataModule,
+    RESISC45DataModule,
+    SEN12MSDataModule,
+    So2SatDataModule,
+    UCMercedDataModule,
+)
+from torchgeo.trainers import (
+    BYOLTask,
+    ClassificationTask,
+    MultiLabelClassificationTask,
+    RegressionTask,
+    SemanticSegmentationTask,
+)
+
+TASK_TO_MODULES_MAPPING: Dict[
```

Comment:
Now that this is in two different scripts, we should move it somewhere where both scripts can find it to avoid code duplication. How about a `torchgeo/common.py` file? I don't want to put it in `torchgeo/__init__.py` because this will be sourced on every import.

Reply:
Like the idea of `torchgeo/common.py`. I can add that in a follow up PR.

Reply:
+1 to this